src/includes/class-wp-cli.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * Network Management Command
+ *
+ */
+class WPMultiNetwork_Command extends WP_CLI_Command {
+
+	/**
+	 * @var array $obj_fields Default fields to display for each object.
```

Comment:
Sorry, just copied this from another command. I still like it, it may get reused, so I like to keep it a variable it possible.